test/system/220-healthcheck.bats

```diff
@@ -487,13 +488,20 @@ function _check_health_log {
     rc=0
     wait -n $hc_pid || rc=$?
     assert $rc -eq 1 "exit status check of healthcheck command"
+    assert $(cat $hcStatus) =~ "stopped" "Health status"
 
     run_podman inspect $ctr --format "{{.State.Status}}"
     assert "$output" == "exited" "Container is stopped"
 
     run_podman inspect $ctr --format "{{.State.Health.Log}}"
     assert "$output" !~ "$msg" "Health log message not found"
 
+    run_podman inspect $ctr --format "{{.State.Health.Status}}"
+    assert "$output" == "stopped" "Health status"
+
+    run_podman inspect $ctr --format "{{.State.Health.FailingStreak}}"
+    assert "$output" == "0"
+
```

Comment:
running podman inspect over and over is slow, you can get all the data in one single inspect command

i.e. see https://github.com/containers/podman/pull/24749#discussion_r1869401866 and https://github.com/containers/podman/pull/22886/commits/8fa1ffbbec60e462073dd3aa6bf09cf910cdb3e2